client/reader/onboarding/style.scss

```diff
@@ -1,30 +1,28 @@
+@import "@wordpress/base-styles/breakpoints";
 .reader-onboarding {
 	display: flex;
-	width: 100%;
 	flex-wrap: wrap;
+	background-color: var(--color-surface-1);
+	border-radius: 6px; /* stylelint-disable-line scales/radii */
```

Comment:
This lint ignore is to match the Reader post cards. I didn't add all of the card styles here, like the bottom border.